flagsmith-rum/README.md

```diff
@@ -0,0 +1,33 @@
+# Agent Check: Flagsmith / Datadog RUM
+
+## Overview
+
+[Flagsmith][1] facilitates feature management across web, mobile, and server side applications.
```

Comment:
Is this meant to be the Flagsmith homepage?

Reply:
OK all these are addressed - thanks!